contributors/devel/development.md

```diff
@@ -134,7 +134,9 @@ development environment, please [set one up](http://golang.org/doc/code.html).
 | 1.5, 1.6       | 1.7 - 1.7.5 |
 | 1.7            | 1.8.1       |
 | 1.8            | 1.8.3       |
-| 1.9+           | 1.9.1       |
+| 1.9            | 1.9.2       |
```

Comment:
this is go1.9.1: https://github.com/kubernetes/kubernetes/blob/release-1.9/hack/lib/golang.sh#L325